eland/utils.py

```diff
@@ -62,7 +61,7 @@ def to_list(x: Union[Collection[Any], pd.Series]) -> List[Any]:
     raise NotImplementedError(f"Could not convert {type(x).__name__} into a list")
 
 
-def try_sort(iterable: Iterable[Item]) -> Iterable[Item]:
+def try_sort(iterable: Iterable[str]) -> Iterable[str]:
```

Comment:
This works for us I believe but I wonder what the general solution to this issue is.

Reply:
I tried looking for solution, Tried a lot of combinations :) But couldnt find any. 😕